doc/command_ref.rst

```diff
@@ -972,10 +972,11 @@ Makecache Command
 
 ``dnf [options] makecache --timer``
     Like plain ``makecache``, but instructs DNF to be more resource-aware,
-    meaning it will not do anything if running on battery power and will terminate
+    meaning it will not do anything if running on battery power, it will terminate
     immediately if it's too soon after the last successful ``makecache`` run
     (see :manpage:`dnf.conf(5)`, :ref:`metadata_timer_sync
-    <metadata_timer_sync-label>`).
+    <metadata_timer_sync-label>`), and it won't try to synchronize the metadata
+    from more mirrors in the mirrorlist if the first one fails.
```

Comment:
This might be a little bit confusing since I think here the mirrorlist refers to all available mirrors but dnf has a specific `mirrorlist` repo config option.

Reply:
Thanks, I updated the description, so that it should be clear that it's per repo.